.github/workflows/ci.yml

```diff
@@ -40,18 +40,23 @@ jobs:
                 args: analyse
 
     ci:
-        name: Test PHP ${{ matrix.php-version }} ${{ matrix.name }}
+        name: Test PHP ${{ matrix.php-version }} & Symfony ${{ matrix.symfony-version }} ${{ matrix.composer-flags }}
         runs-on: ubuntu-latest
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-version }}
         strategy:
             fail-fast: false
             matrix:
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
```

Comment:
it could be removed, since it's not used anymore.

Reply:
What could be removed/is not used anymore?

Reply:
I fixed it in c10c5d68fe42f2d4aef295a255352f8d79eff085

Reply:
My bad, I don't know why I thought your comment was about the highlighted line :D

Reply:
no problem :) again, thanks for the hard work here :) 💛